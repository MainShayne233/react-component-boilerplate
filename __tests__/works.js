import React from 'react'
import renderer from 'react-test-renderer'
import { Counter } from '../src/index.js'

describe('Counter', () => {
  it ('Renders the counter component', () => {
    const component = renderer.create(<Counter/>)
    const json = component.toJSON()
    expect(json).toMatchSnapshot()
  })
})
