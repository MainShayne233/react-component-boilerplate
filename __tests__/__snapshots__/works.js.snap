// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Counter Renders the counter component 1`] = `
<div
  style={
    Object {
      "margin": "0 auto",
    }
  }
>
  <h2>
    Count: 
    0
  </h2>
  <button
    className="btn btn-default"
    onClick={[Function]}
  >
    Increment
  </button>
   
  <button
    className="btn btn-default"
    onClick={[Function]}
  >
    Decrement
  </button>
   
  <button
    className="btn btn-default"
    onClick={[Function]}
  >
    Double (Async)
  </button>
</div>
`;
